editor/editor_properties_array_dict.cpp

```diff
@@ -517,6 +543,59 @@ void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_d
 		emit_changed(get_edited_property(), array, "", false);
 		update_property();
 	}
+
+	if (drag_data.has("type") && String(drag_data["type"]) == "nodes") {
+		Vector<Variant> nodes = drag_data["nodes"];
```

Comment:
You were using Array for the same data above 🤔
If this works, you can replace the loop below to use range iterator (`for (Node *node : nodes)`.

Reply:
> If this works though, you can replace the loop below to use range iterator (for (Node *node : nodes))

I'm not sure how I could do this because right now `nodes` is a `Vector<Variant>` so I would have `no suitable conversion function from "Variant" to "Node *" exists` error 😳

Reply:
Ah right, nevermind then .-.
Though it's weird that it allows `Vector<Variant>` if it's an Array. There might be some conversion happening internally, so you should use the proper type.

Reply:
I used Vector<Variant> because the existing code for handling dropping files resources was using Vector<String> ... Should I change that to Array as well then for consistency ?

Reply:
Yes, it's better to use Array, because that's the actual type of the data.